salt/kubelet/kubelet-config.jinja

```diff
@@ -0,0 +1,61 @@
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+{%- if grains['lsb_distrib_id'] == "CAASP" %}
```

Comment:
I don't think theres any reason to limit this to just CaaSP?

Reply:
Awesome, I wanted to remove it too